### PR TITLE
W-12467261 | Release Edifact connector 2.8.0

### PR DESCRIPTION
**Pull request by sms-ebarbeito**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+| Enforce valid characters | Boolean a| Enforce valid characters when using syntax levels UNOA and UNOB. 
+
+* true
++
+The connector rejects received messages that contain characters not allowed by the syntax level.
++
+* false
++
+The connector either uses a substitution character if one is set, or passes the disallowed characters through. |  true |
+| Use Functional Groups | Boolean a| Version of the group message inside functional group envelopes (UNG - UNE):
```
Comment (sms-dchen) [inline edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc]:
```suggestion
| Use Functional Groups | Boolean a| Group messages with the same functional group identifier inside UNG - UNE envelopes:
```
I believe this description fits better.

**Review by vijayang — DISMISSED** [review]

File: edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+* On the send side, the connector uses this value as the UNB interchange sender identification unless it is overridden in the message parameters. |  |
+//Mule ID qualifier
+| Mule ID qualifier | String a|  Identification qualifier for Mule in the exchange (UNB segment). This value is used in combination with the value of the *Mule interchange ID* field, and if one field is set, the other must also be set. 
+{sp}+
+{sp}+
+If set, this value affects both send and receive message handling:
+
+*  On the receive side, the UNB interchange recipient identification code qualifier must match this value. 
+* On the send side, the connector uses this value as the UNB interchange sender identification code qualifier unless it is overridden in either the send configuration or the message parameters. |  |1
+| Mule application sender/recipient ID | String a|  Identification for Mule in the exchange (UNG segment). This value is used in combination with the value of the *Mule application sender/recipient ID qualifier* field, and if one field is set, the other must also be set.
```
Comment (valkyrie69) [inline edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc]:
```suggestion
| Mule application sender/recipient ID | String a|  Identification for the Mule app in the exchange (UNG segment). This value is used in combination with the value of the *Mule application sender/recipient ID qualifier* field, and if one field is set, the other must also be set.
```

Comment (valkyrie69) [reply edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc]: ? Not sure what "the Mule" is referring to.

File: edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+{sp}+
+{sp}+
+If set, this value affects both send and receive message handling:
+
+* On the receive side, the UNG group recipient identification must match this value.
+* On the send side, the connector uses this value as the UNG group sender identification unless it is overridden in the message parameters. |  |
+| Mule application sender/recipient ID qualifier | String a|  Identification qualifier for Mule in the exchange (UNG segment). This value is used in combination with the value of the *Mule application sender/recipient ID* field, and if one field is set, the other must also be set.
```
Comment (valkyrie69) [inline edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc]: Same comment as above. "the Mule"?
I wonder why the field name isn't showing up as bold.

File: edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,527 @@
...
+* On the send side, the connector uses this value as the UNB interchange recipient identification code qualifier unless it is overridden in either the send configuration or the message parameters. |  |
+| Partner application sender/recipient ID | String a|  Identification for the partner in the exchange (UNG segment). This value is used in combination with the value of the *Partner application sender/recipient ID qualifier* field, and if one field is set, the other must also be set.
+{sp}+
+{sp}+
+If set, this value affects both send and receive message handling:
+
+* On the receive side, the UNG group sender identification must match this value.
+* On the send side, the connector uses this value as the UNG group recipient identification code unless it is overridden in either the send configuration or the message parameters. |  |
+| Parter application sender/recipient ID qualifier | String a|  Identification qualifier for the partner in the exchange (UNG segment). This value is used in combination with the value of the *Partner application sender/recipient ID* field, and if one field is set, the other must also be set.
```
Comment (valkyrie69) [inline edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-reference.adoc]:
```suggestion
| Partner application sender/recipient ID qualifier | String a|  Identification qualifier for the partner in the exchange (UNG segment). This value is used in combination with the value of the *Partner application sender/recipient ID* field, and if one field is set, the other must also be set.
```

File: edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+|requestAcks="false" |Request acknowledgments for Send interchanges using the Acknowledgment Request field (UNB.9).
+|testIndicator="" |Test indicator digit used on Send interchanges (UNB.11). By default interchanges are not sent as tests.
+|useSuppliedValues="false" |Choose your own data values for control-segment identifiers such as `UNB` and `UNZ` segments or use the supplied values. `false` always generates control numbers when writing letting you choose values.
+|outputEdiMimeType="APPLICATION_PLAIN"/"APPLICATION_EDIFACT" |Output MIME type of the message.
+|writeInterchangeControlNumberKey="" |Interchange number key for the object store. If a key is specified, it is used for subsequent operations that use the object store for interchange control numbers.
+|writeGroupControlNumberKey="" |Group control number key for the object store. If a key is specified, it is used for subsequent operations that use the object store for group control numbers.
+|writeMessageControlNumberKey="" |Message control number key for the object store. If a key is specified, it is used for subsequent operations that use the object store for message control number.
```
Comment (valkyrie69) [inline edifact-edi/2.8/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc]:
```suggestion
|writeMessageControlNumberKey="" |Message control number key for the object store. If a key is specified, it is used for subsequent operations that use the object store for message control numbers.
```

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**